reactive/archive_auth_mirror.py

```diff
@@ -1,14 +1,18 @@
 from charmhelpers.core import hookenv
 
-from charms.reactive import when, when_not
+from charms.reactive import when, when_not, hook
 
 from charms.layer.nginx import configure_site
 from charms.archive_auth_mirror import gpg, reprepro, utils
 
 
+@hook('install')
+def install():
+    utils.install_resources()
```

Comment:
Can you show where reactive ensure that @hook handlers get run before @when handlers?